pkg/gridtypes/zos/gw_fqdn.go

```diff
@@ -19,24 +20,41 @@ type Backend string
 
 // check if valid http://ip:port, http://ip or ip:port
 func (b Backend) Valid(tls bool) error {
-	u, err := url.Parse(string(b))
-	if err != nil {
-		return errors.Wrap(err, "failed to parse backend")
-	}
+	var hostName string
 	if tls {
-		if u.Scheme != "" {
-			return fmt.Errorf("scheme expected to be empty")
+		host, port, err := net.SplitHostPort(string(b))
+		if err != nil {
+			return fmt.Errorf("failed to parse backend with error: %w", err)
+		}
+		if port == "" {
```

Comment:
This is not necessary, if you check the code for the SplitHostPort function you see that it already return error if the format is wrong or if the port is empty.

But it does validate that the IP value or the Port value are valid. hence parsing of the splited ip, and port still need to be done of course